manifests/profiles/PROFILES.md

```diff
@@ -10,7 +10,7 @@ will provide further instructions. Additionally, please read the RFC requirement
 
 ## Vendor optimized profiles
 
-| K8s Platform         | Vendor        | Lead Maintainer | Lead Maintainer  |
-| -------------------- | ------------- | --------------- | ---------------- |
-| OpenShift            | Red Hat / IBM | @jwendell       | @irisdingbj      |
-| Upstream Kubernetes  | Istio         | Env WG          | discuss.istio.io |
+|Profile Name| K8s Platform         | Vendor        | Lead Maintainer | Lead Maintainer  |
+|------------| -------------------- | ------------- | --------------- | ---------------- |
+|openshift| OpenShift            | Red Hat / IBM | @jwendell       | @irisdingbj      |
+|| Upstream Kubernetes  | Istio         | Env WG          | discuss.istio.io |
```

Comment:
extra `|` doesnt' seem to make sense to me.

Reply:
Added the extra `|` here because there's no profile name for the bottom one. After reading the comment above, perhaps it's better to say "see list at istio.io" or something?